css-backgrounds-4/Overview.bs

```diff
@@ -771,16 +771,28 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animatable: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animatable: by computed value,
```

Comment:
Can you please use `Animation type`, as required in [Web Animations](https://drafts.csswg.org/web-animations-1/#animation-type)? 🙏 Otherwise `w3c/reffy` would extract rules with fields mixing `animationType`/`animatable`.

Reply:
Ah sorry! https://github.com/w3c/csswg-drafts/pull/8605#discussion_r1141162192